website/docs/topics/the-anatomy-of-unleash.mdx

```diff
@@ -0,0 +1,195 @@
+
+Unleash comes with a number of strategies built in (refer the [activation strategies documentation](../user_guide/activation-strategies.md) for more information on those). You can also create your own [custom activation strategies](../advanced/custom-activation-strategy.md) if you need them. All strategies can be further augmented by [**strategy constraints**](../advanced/strategy-constraints.md).
+
+
+<Figure img="/img/anatomy-of-unleash-environments-strategies.png" caption="Feature toggles exist across environments and can have different activation strategies in each environment."/>
+
+<Figure img="/img/anatomy-of-unleash-environments-strategies2.png" caption="Feature toggle activation strategies can be copied between environments. You can also create new strategies in each environment."/>
+
+## Strategy constraints
+
+[**Strategy constraints**](../advanced/strategy-constraints.md) (or just **constraints**) help you fine-tune your strategies. They are an extra layer of prerequisites that help you narrow the audience of a strategy down. Strategy constraints are applied to [**activation strategies**](../user_guide/activation-strategies.md)
+
+For example, if you wanted to roll a feature out to 50% of users with **a specific email domain **(such as “@mycompany.com”), then strategy constraints would let you target only users with that email domain.
+
+Constraints can also be used for more general purposes, such as timing feature releases or releasing features in specific regions.
+
+An activation strategy can have as many constraints as you want. When an activation strategy has multiple constraints, then **every constraint **must be satisfied for the strategy to be evaluated. So if you have two constraints: one that says users must have an “@mycompany.com” email address and one that says users must have signed up for a beta program, then the strategy would **only be evaluated for users with @mycompany.com emails that have signed up for the program**.
+
+:::tip Strategies and constraints
+
+Feature toggle strategies are **permissive**: As long as **one** strategy resolves to true, the feature is considered enabled. On the other hand, constrains are **restrictive**: for a given strategy, **all** constraints must be met for it to resolve to true.
+
+We can exemplify this difference with the logical operators AND and OR:
+- For a feature toggle, if strategy 1 OR strategy 2 OR .. OR strategy `n` is true, **then the feature is enabled**.
+- For a strategy, it can be evaluated **if and only if** constraint 1 AND constraint 2 AND .. AND constraint `n` are met.
+
+Note that even if all the constraints are met, the strategy itself might not resolve to true: that will depend on the strategy and the provided context.
```

Comment:
Well, some strategies use the context to evaluate, right? So for those strategies, the provided context matters. But I agree that we might be able to clarify it somehow.

Reply:
Yeah, things like gradual rollout will for stickiness reasons. It's more that a constraint will never execute properly without a context (honestly I haven't actually tried any of the SDKs with a no context against a toggle with constraints but it will do something weird).

Looking at this with fresh eyes in the morning this looks fine to me